### ISSUE TITLE
The `role` attribute is not defined in the `attr-order` rule

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#4033]
- Markuplint version: `5.0.0-rc.7`
- Parser lang: HTML
- Node.js version: `24.17.0`
- OS: Windows

## Describe the bug

バージョン5で追加される `attr-order` ルールに `role` 属性の定義が抜けているようです。

https://github.com/markuplint/markuplint/blob/37dbcef8830e0eeaa95ac2bfe92f18a7f2418fb5/packages/%40markuplint/rules/src/attr-order/index.ts#L168-L174

ここに文字列 `role` の判定を追加すればとりあえず解消するかもしれませんが、個人的には `role` と `aria-*` のどちらを優先するかもカスタマイズできた方が嬉しいです。`aria` グループとは別に `role` グループ(?)を追加することは可能でしょうか。

## Code Example

```js
export default {
	rules: {
		'attr-order': [
			{ group: 'global', order: 'source-order' },
			{ group: 'aria', order: 'source-order' },
		],
	},
};

```

```html
<ul role="radiogroup" aria-label="XXX"></ul>
```

## The current behavior

> warning: "aria-label"は"role"より前にしたほうがよいです (attr-order)

## The expected behavior

エラーなし
